server/pb_hooks/pages/index.ejs

```diff
@@ -4,4 +4,173 @@
+      <li>2.75</li>
+      <li>3.50</li>
+      <li>4.25</li>
+      <li>4.00</li>
+      <li>4.50</li>
+      <li>4.50</li>
+      <li>4.50</li>
+      <li>4.25</li>
+      <li>4.50</li>
+
+      <li>4.50</li>
+      <li>4.50</li>
+      <li>3.00</li>
+    </ul>
+    <!-- medium prices -->
+    <ul class="col-span-1">
+      <li>3.25</li>
+      <li>3.25</li>
+      <li>4.00</li>
+      <li>4.75</li>
+      <li>4.25</li>
+      <li>5.00</li>
+      <li>5.00</li>
+      <li>5.00</li>
+      <li>4.75</li>
+      <li>5.00</li>
+
+      <li>4.25</li>
+      <li>5.00</li>
+      <li>3.50</li>
+    </ul>
+    <!-- large prices -->
+    <ul class="col-span-1">
+      <li>3.75</li>
+      <li>3.75</li>
+      <li>4.00</li>
+      <li>5.25</li>
+      <li>4.75</li>
+      <li>5.50</li>
+      <li>5.50</li>
+      <li>5.50</li>
+      <li>5.25</li>
+      <li>5.50</li>
+
+      <li>4.75</li>
+      <li>5.50</li>
+      <li>4.00</li>
+    </ul>
+  </div>
+  <!-- Drink Sizes -->
+  <div class="grid grid-cols-3 col-span-1 pb-6">
+    <h3 class="text-3xl pl-0 col-span-full">Drink Sizes</h3>
+    <p class="text-3xl">12</p>
+    <p class="text-3xl">16</p>
+    <p class="text-3xl">20</p>
+    <p>oz</p>
+    <p>oz</p>
+    <p>oz</p>
+    <p>Small</p>
+    <p>Medium</p>
+    <p>Large</p>
+    <p class="text-xl">Hot</p>
+    <p>or</p>
+    <p class="text-xl">Iced</p>
+  </div>
+  <!-- Drip Brew -->
+  <div class="grid grid-cols-3 col-span-1 pb-6">
+    <h3 class="text-3xl pl-0 col-span-full">Drip Brew</h3>
+    <p>3.00</p>
+    <p>3.50</p>
+    <p>4.25</p>
+    <p class="col-span-full">8oz Mug</p>
+    <p class="col-span-2">Unlimited</p>
+    <p>3.00</p>
+  </div>
+</div>
+<!-- Menu Grid -->
+<div class="grid grid-cols-1 md:grid-cols-2">
+  <div class="menu col-span-full pl-0">
+    <h2 class="text-6xl pl-0">Menu</h2>
+  </div>  
+  <div class="quiche row-span-2 pb-6">
+    <h3 class="text-2xl">Quiche: $7.50</h3>
+    <ul>
+      <li>
+        Broccoli Cheddar
+      </li>
+    </ul>
+  </div>
+  <div class="breakfast-sandwich pb-6">
+    <h3 class="text-2xl">Breakfast Sandwich: $9.25</h3>
+    <p>On sourdough bread</p>
+    <div class="grid grid-cols-1 md:grid-cols-3">
+      <p>Veggie Sausage Egg and Cheddar</p>
+      <p class="text-center">OR</p>
+      <p>Ham Egg and Swiss</p>
+    </div>
+  </div>
+  <div class="breakfast-burrito pb-6">
+    <h3 class="text-2xl">Breakfast Burrito: $7.75</h3>
+    <p>
+      Potato, scrambled eggs, veggies, monterey jack cheese
+      and sour cream served with ranchero sauce and corn
+      chips
+    </p>
+  </div>
+  <div class="turkey">
+    <h3 class="text-2xl">Turkey Provolone Sandwich: $9.25</h3>
+  </div>
+  <div class="black-bean-burrito">
+    <h3 class="text-2xl">Black Bean Burrito: $7.00</h3>
+    <p>
+      Seasoned black beans, jasmine rice and monterey jack
+      cheese <br/>
+      ❤️ Vegan option available ❤️ <br/>
+      Served with corn chips and salsa
+    </p>
+  </div>
+</div>
+<!--  
```

Comment:
Ignore this commented block, I just didn't want to retype the teas out after my first transcription of them